sdk/examples/wallet/threads.rs

```diff
@@ -7,7 +7,7 @@
 
 use std::env;
 
-use dotenv::dotenv;
+use dotenvy::dotenv;
```

Comment:
Bit annoying I know, but can I ask you to remove imports of dotenvy and use it scoped when needed please? Maybe even std::env if you feel like it. Like the client examples. In the examples I think it's better to explicitly import only wallet/client related stuff since it's show in the wiki, to distract readers a bit less.

Reply:
Just checking you didn't forget do push or whatever. Will you do this?

Reply:
yeah, doing it!